media/css/m24/components/footer-newsletter.scss

```diff
@@ -209,25 +226,10 @@ $max-footer-content-width: $content-max;
 
             @media #{$mq-lg} {
                 display: inline-grid;
-
-                &:first-of-type {
-                    grid-column-start: 1;
-                    grid-column-end: 2;
-                }
-
-                &:nth-of-type(2) {
-                    grid-column-start: 2;
-                    grid-column-end: 3;
-                }
             }
 
             &:has(> label.mzp-u-inline) {
-                margin: $spacer-lg 0 $spacer-md;
-
-                @media #{$mq-lg} {
```

Comment:
made full width the column default for children of newsletter details above mq-lg